calico/reference/resources/profile.md

```diff
@@ -9,7 +9,7 @@ policy rules, but that feature is deprecated in favor of the much more
 flexible [NetworkPolicy]({{ site.baseurl }}/reference/resources/networkpolicy) and
 [GlobalNetworkPolicy]({{ site.baseurl }}/reference/resources/globalnetworkpolicy) resources.
 
-Each {{site.prodname}} endpoint or host endpoint can be assigned to zero or more profiles.
+Each {{site.prodname}} endpoint or host endpoint can be assigned to zero or more profiles. {{site.prodname}} provides a default profile resource named `projectcalico-default-allow` that consists of allow-all ingress and egress rules. Adding that profile to host endpoints will change the default behaviour of traffic originating from or terminating at the host. See the [host endpoint reference]({{site.baseurl}}/reference/resources/hostendpoint#default-behavior-of-external-traffic-tofrom-host) for more information.
```

Comment:
On reflection it feels odd that we chose to implement this with a profile, when I think we're also deprecating the use of profile rules.

But perhaps we think it's OK for **us** to use profiles to get a certain behaviour, but it's deprecated for users.

If that's the case, it would be better to deemphasize the detail of how this works, and just describe the effect:
> provides a profile resource named `projectcalico-default-allow`, and if you add that profile to a host endpoint, traffic originating from or terminating at the host will be allowed by default (i.e. if there is no applicable policy that denies the traffic first), rather than denied. See the ...